Dockerfile

```diff
@@ -1,22 +1,29 @@
-FROM debian:jessie
+FROM buildpack-deps:jessie-scm
 MAINTAINER Rob Hoelz
 
-ENV rakudo_version=2015.03
-
 RUN groupadd -r perl6 && useradd -r -g perl6 perl6
 
-ADD http://rakudo.org/downloads/star/rakudo-star-${rakudo_version}.tar.gz /root/
-RUN tar xzf /root/rakudo-star-${rakudo_version}.tar.gz -C /root/
-
-RUN apt-get update && apt-get --yes install \
-    gcc \
-    make \
-    perl-modules
-
-WORKDIR /root/rakudo-star-${rakudo_version}
+ENV rakudo_version=2015.03
 
-RUN perl Configure.pl --prefix=/usr --gen-moar
-RUN make install
+RUN buildDeps=' \
+        gcc \
+        libc6-dev \
+        libencode-perl \
+        make \
```

Comment:
I think that enough of those are required by the modules in Rakudo Star that they should be left in.  I'll try out all of the modules to see if other relevant libraries are missing.